www/FabricPlugin.Crashlytics.js

```diff
@@ -1,67 +1,51 @@
-'use strict';
 
```

Comment:
This file should be JavaScript, but it now contains es6 code. It looks like the something went wrong while transpiling the code. This breaks the entire plugin on iOS.